amundsen_application/static/js/components/ColumnList/index.tsx

```diff
@@ -76,6 +90,50 @@ type ExpandedRowProps = {
 };
 
 const SHOW_STATS_THRESHOLD = 1;
+const USAGE_STAT_TYPE = 'column_usage';
```

Comment:
This would probably be the item to move to the config, however I am wary of creating that abstraction now, without knowing more about what it is to come.

Reply:
I can understand that. I am alright with it as it since the community will be able to follow existing examples to make this more flexible if/when needed.